plugins/java-sdk-code-completion-plugin/README.md

```diff
@@ -0,0 +1,43 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+# Apache Beam Java SDK Completion Plugin
+
+The Beam Java SDK Completion Plugin is an IntelliJ Plugin that provides Java SDK Beam Transform completions.
```

Comment:
can you add information about the Google Summer of Code program, and link the proposal document?

Please also specify that this is an experimental community project, and is not officially supported by the Beam community.

Finally, also mention somewhere that the project builds with Gradle, but that it's not part of the rest of Beam Gradle configuration because this is intended to be an isolated community project. (perhaps add a ` ## Development` section with that information.